models/public/aclnet-int8/aclnet-int8.md

```diff
@@ -0,0 +1,67 @@
+# aclnet-int8
+
+## Use Case and High-Level Description
+
+The `AclNet-int8` model is quantized and fine-tuned with NNCF variant of [AclNet](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/aclnet/aclnet.md) model, which is designed to perform sound classification.
+The `AclNet-int8` model is trained on an internal dataset of environmental sounds for 53 different [classes](https://github.com/openvinotoolkit/open_model_zoo/tree/develop/data/dataset_classes/aclnet_classes.txt).
+For details about the model, see this [paper](https://arxiv.org/abs/1811.06669).
+
+The model input is a segment of PCM audio samples in [N, C, 1, L] format.
+
+The model output for `AclNet-int8` is the sound classifier output for the 53 different environmental sound classes from the internal sound database.
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 2.71          |
+| MParams           | 1.41          |
+| Source framework  | PyTorch\*     |
+
+## Accuracy
+
+| Metric | Value |
+| ------ | ----- |
+| Top 1  | 87.1% |
+| Top 5  | 93.0% |
+
+Metrics were computed on internal validation dataset according to following [publication](http://dcase.community/documents/workshop2019/proceedings/DCASE2019Workshop_Huang_52.pdf) and [paper](https://arxiv.org/abs/1811.06669).
+
+## Input
+
+### Original Model
+
+Audio, name - `result.1`, shape - `1,1,1,L`, format is `N,C,1,L` where:
+
+- `N` - batch size
+- `C` - channel
+- `L` - number of PCM samples (minimum value is 16000)
+
+### Converted Model
+
+Audio, name - `result.1`, shape - `1,1,1,L`, format is `N,C,1,L` where:
+
+- `N` - batch size
+- `C` - channel
+- `L` - number of PCM samples (minimum value is 16000)
+
+## Output
+
+### Original Model
+
+Sound classifier (see [labels](./labels.txt)), name - `486`, shape - `1,53`, output data format is `N,C` where:
```

Comment:
```suggestion
Sound classifier (see [labels](../../../data/dataset_classes/aclnet_classes.txt)), name - `486`, shape - `1,53`, output data format is `N,C` where:
```

Reply:
I would actually suggest to write `OMZ_ROOT/data/dataset_classes/...`. Links to non-Markdown files don't work in the published documentation.

Reply:
this should be done globally for OMZ, so better to do in separate PR